src/Drupal/Driver/DrushDriver.php

```diff
@@ -189,11 +189,27 @@ public function clearStaticCaches() {
   }
 
   /**
+   * Decodes JSON output returned by Drush.
+   *
+   * It will clean up any junk that may have appeared before the JSON.
+   *
+   * @param string $output
+   *   The output from Drush.
+   * @return object
+   *   The decoded JSON object.
+   */
+  protected function decodeJsonOutput($output) {
+    // Remove anything before the first '{'.
+    $output = preg_replace('/^[^\{]*/', '', $output);
```

Comment:
I don't think the 's' modifier is needed because we aren't using `.` in the regex. The docs say:

> A negative class such as [^a] always matches a newline character, independent of the setting of this modifier.

However, that still looks like to me like it'll match up until the first `}` rather than the last `}`, but I'd be happy to be wrong. :-)